manta-benchmark/www/index.js

```diff
@@ -2,6 +2,7 @@ import { Context, Proof, prove_mint, prove_private_transfer, prove_reclaim, veri
 
 const pre = document.getElementById("wasm-prover");
 
+const REPEAT = 5;
```

Comment:
We should have different values for `prove` and `verify` because `verify` takes much less time so we can take more samples.